_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -4695,6 +4749,52 @@ mfxStatus ValidateParams(mfxVideoParam *par, mfxVppCaps *caps, VideoCORE *core,
 
         switch(id)
         {
+        case MFX_EXTBUFF_VPP_MIRRORING:
+        {
+            mfxExtVPPMirroring* extMir = (mfxExtVPPMirroring*)data;
+            // SW mirroring supports only horizontal mode
+            mfxU32 maxMirrorSupportMode = 1;
```

Comment:
what is '1' and '2'? Please use named constants

Reply:
not addressed.

Reply:
fixed